stdlib/ext/file-ext.mc

```diff
@@ -105,6 +111,18 @@ utest
   else ("Error reading file","","","")
 with ("Hello", "Next string", "Final", "EOF") in
 
+-- Test reading x amount of characters from the file
```

Comment:
Consider adding a test case for the case in which `None ()` is returned

Reply:
Fixed!